northstar/src/runtime/ipc/channel.rs

```diff
@@ -1,93 +1,194 @@
-use std::{
-    io::{ErrorKind, Read, Write},
-    os::unix::prelude::{AsRawFd, RawFd},
-};
+use std::io::{Read, Write};
```

Comment:
This file has mixed style trait bound definitions. I'd just one bound everywhere so it's probably good to declare that with the `impl` instead of a `where` clause.

Reply:
now that you mention it... :rofl: yeah, will try to make it more consistent